modules/tools/pages/query-workbench.adoc

```diff
@@ -1,4 +1,5 @@
 = Query Workbench
+:page-aliases: developer-guide:query-workbench-intro
```

Comment:
This needs to be fixed in 4.1 as well. I assume that a PR for 4.1 is coming?

Reply:
Created #485 for release/4.1